src/transformers/modeling_attn_mask_utils.py

```diff
@@ -199,80 +200,11 @@ def _unmask_unattended(
         `attention_mask` is [bsz, src_seq_len].
 
         The dimension num_masks of `expanded_mask` is most often 1, but it can also be the number of heads in the case of alibi attention bias.
-
```

Comment:
the comment was still a bit helpfull IMO

Reply:
Agreed! If it's not completely relevant any more can I be updated?

Reply:
Yes it should not have been removed, added it back